index.js

```diff
@@ -29,7 +29,15 @@ ResourceManifestPlugin.prototype.apply = function(compiler) {
 
 function format(assets, path) {
   return `
-ui_page "${path}/index.html"
+  name 'resource name'
```

Comment:
Can you remove the indentation please? It causes the generated manifest to have indentations too.